downstream/modules/platform/con-controller-capacity-determination.adoc

```diff
@@ -0,0 +1,28 @@
+[id="controller-capacity-determination"]
+
+= {ControllerNameStart} capacity determination and job impact
+
+Learn how to determine capacity for instance groups and its impact on your jobs. 
+
+{ControllerNameStart} capacity system determines how many jobs can run on an instance given the amount of resources available to the instance and the size of the jobs that are running (referred to as Impact). 
```

Comment:
If the capacity system is part of Controller,then it's The {ControllerName} capacity system